add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/java/dev/flutter/multipleflutters/MainActivity.kt

```diff
@@ -0,0 +1,56 @@
+package dev.flutter.multipleflutters
+
+import android.app.ActivityManager
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import io.flutter.embedding.android.FlutterActivity
+
+/**
+ * The activity that uses standard platform UI APIs.
+ *
+ * This doesn't talk directly to or render Flutter content and represents a part of your app that
+ * would potentially already be written that wants to interface with Flutter.
+ */
+class MainActivity : AppCompatActivity(), DataModelObserver {
```

Comment:
I like the swift name better. It's not so much of a "main" activity since it'll cycle back over and over again.

Reply:
There's still this bit (maybe you didn't want to do a rename in the same commit)